src/main/java/core/basesyntax/service/impl/CsvDataParser.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.ReadDataParser;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsvDataParser implements ReadDataParser {
+    private static final String LINE_SPLIT_REGEX = ",";
+    private static final int OPERATION_TYPE_CODE_INDEX = 0;
+    private static final int PRODUCT_TYPE_INDEX = 1;
+    private static final int AMOUNT_INDEX = 2;
+
+    @Override
+    public List<FruitTransaction> parseToTransactionList(List<String> data) {
+        int firsDataLineIndex = 1;
+        List<FruitTransaction> resultList = new ArrayList<>();
+
+        for (int i = firsDataLineIndex; i < data.size(); i++) {
+            String[] splitLine = data.get(i).split(LINE_SPLIT_REGEX);
+            resultList.add(new FruitTransaction(parseOperationType(splitLine),
+                    parseProductType(splitLine), parseAmount(splitLine)));
+        }

```

Comment:
OPTIONAL: I think it is possible to do it using stream and use method .skip, you will avoid creating firsDataLineIndex  variable, and code will be more readable